spec/controllers/admin_stats_controller_spec.rb

```diff
@@ -98,5 +98,32 @@
       end
     end
 
+    describe "depositor counts" do
+      before do
+        GenericFile.new(id: "abc123") do |gf|
```

Comment:
@mjgiarlo Yes the ids needs to be included.  I'm cheating the system a bit to make the test faster.  Instead of saving to fedora I am only updating the solr index.  Without the id solr is sad.